.github/workflows/build.yml

```
@@ -22,7 +22,7 @@ jobs:
       - name: Build with node
         uses: actions/setup-node@v3
         with:
-          node-version: 18.12.1
+          node-version: 18.16.0
```

If we swap these `node-version` arguments out for [`node-version-file`](https://github.com/actions/setup-node/blob/869f4dd0c7f320ae834c2724d92a364de3893c24/action.yml#L10-L11), then there's just one place to update for all the GitHub actions, instead of three - and no chance of them getting out of sync:

```suggestion
          node-version-file: ".node-version"
```

---

I like this approach because I like setting the exact version everywhere, but earlier [we decided to use the `.node-version` as a major-version social signal to devs rather than drive code and pipelines](https://github.com/PhilanthropyDataCommons/service/issues/134). So I'm not comfortable making the change without consent from @slifty.